### PR TITLE
eventcalendar format / added clicktarget parameter

### PR DESCRIPTION
**Pull request by FelixAba**
to allow e.g. opening a form in edit mode when clicking on an calendar day. See commit messages for details.

### CODE REVIEW

**Comment by kghbln** [comment]
@FelixAba Thank you for authoring the pull request. I have requested reviews for it.

**Comment by kghbln** [comment]
@FelixAba I18n is regularly only added for "en" as the source language and for "qqq" message documentation. All other translations roll in via the translatewiki.net translation platform which is used to provide them. This is however not a holdup since I can add the "qqq" directly other there.

File: formats/calendar/resources/ext.srf.formats.eventcalendar.js [hunk 1]
```diff
@@ -467,6 +471,30 @@
 							// If the day number (where available) is clicked then switch to the daily view
 							if ( allDay && data.query.ask.parameters.dayview && $( jsEvent.target ).is( 'div.fc-day-number' ) ) {
 								container.fullCalendar( 'changeView', 'agendaDay'/* or 'basicDay' */).fullCalendar( 'gotoDate', date );
+							} else {
```
Comment (mwjames) [inline formats/calendar/resources/ext.srf.formats.eventcalendar.js]: Would be nice if you could make this block a separate function (for more cleanliness) and call it here instead.

Comment (FelixAba) [inline formats/calendar/resources/ext.srf.formats.eventcalendar.js]: Hello  mwjames,
Thanks for his suggestion about the function. 
I am  about to rework a few things and will let you people know, when the Pull Request is ready to be merged.

Comment (FelixAba) [inline formats/calendar/resources/ext.srf.formats.eventcalendar.js]: @kghbln  , @mwjames 
 I have built a separate function and now wait for a Review.
My Regards

**Comment by mwjames** [comment]
Please be so kind and verify the change by creating an example at [0] and link it to this issue. Thanks.

[0] https://sandbox.semantic-mediawiki.org

**Comment by kghbln** [comment]
> Please be so kind and verify the change by creating an example at [0] and link it to this issue. Thanks.

@FelixAba This is a weak holdup for the upcoming release. Will be cool. Thanks!

**Comment by s7eph4n** [comment]
@FelixAba, could you document the new parameter on https://www.semantic-mediawiki.org/wiki/Help:Eventcalendar_format?

**Comment by gesinn-it** [comment]
@s7eph4n it's already documented. Is something missing from your point of view?

**Comment by s7eph4n** [comment]
Indeed! Sorry, I missed it.

**Comment by mwjames** [comment]
>  Is something missing from your point of view?

I vaguely remembered me asking for "Please be so kind and verify the change by creating an example at [0] and link it to this issue" but maybe I just missed the reply on this.

**Comment by gesinn-it** [comment]
@mwjames: created an example [0], added to examples in [1]. Simply click on a day in the calendar...

[0] https://sandbox.semantic-mediawiki.org/wiki/Issue/200_(SRF_-_eventcalendar_clicktarget_parameter)
[1] https://www.semantic-mediawiki.org/wiki/Help:Eventcalendar_format

**Pull request merged**